IndicoToolkit/Association/Positioning.cs

```diff
@@ -0,0 +1,321 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+
+using IndicoToolkit.Types;
+using IndicoToolkit.Exception;
+
+namespace IndicoToolkit.Association
+{
+    /// <summary>
+    /// Class <c>Positioning</c> helps identify relative positions in a document using bounding box data.
+    /// Positions are expected to contain, at a minimum, the following top-level keys: "bbTop", "bbBot", "bbLeft", "bbRight", "page_num".
+    /// </summary>
+    public class Positioning {
+        
+        public Positioning() { }
+
+        /// <summary>
+        /// Check if the location of one box is above another.
+        /// </summary>
+        /// <param name="abovePos">The position expected to be above</param>
+        /// <param name="belowPos">The position expected to be below</param>
+        /// <param name="mustBeSamePage">Required to be on same page. Defaults to true.</param>
+        public bool positionedAbove(Position abovePos, Position belowPos, bool mustBeSamePage = true)
+        {
+            bool isAbove = false;
+            if (belowPos.pageNum < abovePos.pageNum)
+            {
+                isAbove = false;
+            }
+            else if (belowPos.pageNum != abovePos.pageNum && mustBeSamePage)
+            {
+                isAbove = false;
+            }
+            else if (xAxisOverlap(abovePos, belowPos))
+            {
+                if (belowPos.pageNum == abovePos.pageNum && !yAxisAbove(abovePos, belowPos))
+                {
+                    isAbove = false;
+                }
+                else
+                {
+                    isAbove = true;
+                }
+            }
+            return isAbove;
+        }
+
+        /// <summary>
+        /// Check if the location of one box is on the same page and above another if the lower box's overlap is at least the given percentage.
```

Comment:
I think we need a better description that this-- it's a little confusing. 

Check if the location of one box is on the same page and above another and that the box's x axis positions overlap by at least "miOverlapPercent"

or something like that